wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,77 @@
+# How to install drivers and fix them 
+Hello, in this post you will know how to download or update your drivers for your computer.
+It's highly recommended to not use third-party software to update drivers.
+## Before you get into the process, you need to know your computer.
+The first step in installing drivers is to correctly identify the hardware you need drivers for. This information can be found in your computer's manual or by using the Device Manager (Windows) (right click into windows icon and press “Device Manager”). In the Device Manager, look for the device you require drivers for, and expand its entry to see its model number. In System Information, navigate to the “Hardware” section and find the device you require drivers for.
+
+## Method 1: Check Windows Update
+You can usually find drivers and drivers update in Windows Update, instead of manually downloading the file.
+ 1. Open the Settings app using the Start menu.
+ 2. Go to Updates and security.
+ 3. Click on “Check updates”.
+ 4. Windows Update might say that you are up-to-date, even if optional updates are available. In this case, you will see an “Optional Updates” button. Clicking on it will show a list of updates (all of them are typically driver updates). Select all of them and click on “Download and install”.
+
+## Method 2: Download via brand device page.
+It's highly recommended to download or update the drivers, through the manufacturer's website, as it's guaranteed that they will work correctly. This is especially a good idea for the GPU (Graphics) drivers.
+All you need to do is:
+
+ 1. On the search engine of your choice (Google, Bing, etc…), search for "{device manufacturer} drivers".
+ 2. Next, enter the ID of the device and follow the instructions on the page.
+ 3. When you selected the product, you have two options:
+	- Download the driver downloader (recommended)
+	- And download the driver on specific and run executable
+
+## Method 3: Download via brand product page.
+Now if you want to download for your GPU or your brand device hasn't the drivers (in this case do it if it is especially important, otherwise apply the saying: “if it works, don't touch it, or you may make it worse”).
+Now follow the same instructions but the brand hardware (like intel CPU and GPU).
+
+ 1. On the search engine of your choice (Google, Bing, etc…), search for "{your brand device name} drivers".
+ 2. Search your hardware piece you want to upgrade.
+ 3. When you select the product, you have two options:
+	- It's recommended to use the driver downloader if it's provided, as it makes the installation a lot easier.
+	- Download the driver on specific and run the executable
+- (Follow method 3 for more information: [https://msft.chat/wiki/reinstalling-gpu-drivers.html#method-3-oem-website](https://msft.chat/wiki/reinstalling-gpu-drivers.html#method-3-oem-website))
```

Comment:
`See [Reinstalling GPU drivers from the OEM's website](https://msft.chat/wiki/reinstalling-gpu-drivers.html#method-3-oem-website) for more information.`